riddlc/src/test/scala/com/reactific/riddl/example.riddl

```diff
@@ -0,0 +1,29 @@
+domain Example is {
```

Comment:
Like the example.conf file, this file is in the wrong location. Please put it in src/test/input